nico/backends/sdl2.nim

```diff
@@ -1829,9 +1829,10 @@ proc getRecordSeconds*(): int =
     return 0
 
 when defined(emscripten):
+  type cbool* {.importc: "bool", nodecl.} = uint8
```

Comment:
```suggestion
  type cbool* {.importc: "bool", header:"<stdbool>".} = uint8
```

This will fail if stdbool is not imported, nodecl is used for compiler builtin, though it's possible that the compiler auto-imports stdbool and issues a warning. See https://www.open-std.org/JTC1/SC22/WG14/www/docs/n1256.pdf p253
![image](https://github.com/user-attachments/assets/63d05e38-3fa7-4547-a6d7-241ecebac0ca)

Reply:
Tested both variants. Only the first one worked on my Windows machine.